src/com/lftechnology/java/training/srijan/FirstTutorial.java

```diff
@@ -0,0 +1,30 @@
+package com.lftechnology.java.training.srijan;
+
+import java.util.Scanner;
+
+public class FirstTutorial {
+public static void main(String[] args){
+		
+		Scanner input = null;
+		
+		try{
```

Comment:
you can use try-with-resources feature to automatically close the opened resources.
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
http://www.mkyong.com/java/try-with-resources-example-in-jdk-7/